magni_bringup/scripts/launch_core.py

```diff
@@ -275,48 +296,41 @@ def main():
     camera_extr_file = ""
     lidar_extr_file = ""
 
-    # this may happen with older robot.yaml where conf["raspicam"] did not have camera_installed entry yet
-    if not "camera_installed" in conf["raspicam"]:
-        print("ERROR: 'camera_installed' entry was not found in conf['raspicam']")
-        return
-
     # check for camera extrinsics
-    if (
-        conf["raspicam"]["camera_installed"] == "True"
-        or conf["raspicam"]["camera_installed"] == "true"
-    ):
+    if (conf["raspicam_position"] != None and conf["raspicam_position"] != "None"):
         # get camera extrinsics
         path1 = (
-            "~/.ros/extrinsics/camera_extrinsics_%s.yaml" % conf["raspicam"]["position"]
+            "~/.ros/extrinsics/camera_extrinsics_%s.yaml" % conf["raspicam_position"]
```

Comment:
has somebody thought through this 'symantic' change?     It only 'slightly' makes my 'spidy sense' thing that there is room for generation of a bug in this sort of synantic change.    

This change MUST be tested by our test group.   Please include a test in the test plan.

Reply:
Not tested with our test group yet.

If the user sets

`raspicam: {"position" : "forward"} 
`

instead of 

`raspicam_position: 'forward'`

The raspicam_position forward will be taken from `src/magni_robot/magni_bringup/config/default_robot.yaml` thus making the robot not chrash, just the cam position will not be what user has expected.

Maybe also here i could add a comment in `/etc/ubiquity/robot.yaml` that the param has changed from `raspicam: {"position" : "forward"}`  to `raspicam_position: 'forward'`. Same thing then needs to be done for lidar